jenkins/scripts/integration_test.sh

```diff
@@ -143,30 +145,36 @@ EPHEMERAL_TEST="${EPHEMERAL_TEST}"
 IRONIC_FROM_SOURCE="${IRONIC_FROM_SOURCE}"
 IRONIC_LOCAL_IMAGE_BRANCH="${IRONIC_LOCAL_IMAGE_BRANCH}"
 IRONIC_LOCAL_IMAGE="${IRONIC_LOCAL_IMAGE}"
+GINKGO_FOCUS="${GINKGO_FOCUS}"
 EOF
 
 # Only set these variables if they actually have values.
 # If the variable is unset or empty (""), do nothing.
 if [[ ! -z "${CAPIRELEASE:+x}" ]]
 then
-  echo "CAPIRELEASE=${CAPIRELEASE}" | tee --append "${CI_DIR}/files/vars.sh"
+  echo "CAPIRELEASE=${CAPIRELEASE}" | tee --append "${CI_DIR}/files/${TEMP_FILE_NAME}"
 fi
 if [[ ! -z "${CAPM3RELEASE:+x}" ]]
 then
-  echo "CAPM3RELEASE=${CAPM3RELEASE}" | tee --append "${CI_DIR}/files/vars.sh"
+  echo "CAPM3RELEASE=${CAPM3RELEASE}" | tee --append "${CI_DIR}/files/${TEMP_FILE_NAME}"
 fi
 
-
-cat "${CI_DIR}/integration_test_env.sh" >> "${CI_DIR}/files/vars.sh"
+cat "${CI_DIR}/integration_test_env.sh" >> "${CI_DIR}/files/${TEMP_FILE_NAME}"
 
 # Send Remote script to Executer
 scp \
   -o StrictHostKeyChecking=no \
   -o UserKnownHostsFile=/dev/null \
   -i "${METAL3_CI_USER_KEY}" \
   "${CI_DIR}/files/run_integration_tests.sh" \
-  "${CI_DIR}/files/vars.sh" \
-  "${METAL3_CI_USER}@${TEST_EXECUTER_IP}:/tmp/" > /dev/null
+  "${METAL3_CI_USER}@${TEST_EXECUTER_IP}:/tmp" > /dev/null
+
+scp \
+  -o StrictHostKeyChecking=no \
+  -o UserKnownHostsFile=/dev/null \
+  -i "${METAL3_CI_USER_KEY}" \
+  "${CI_DIR}/files/${TEMP_FILE_NAME}" \
+  "${METAL3_CI_USER}@${TEST_EXECUTER_IP}:/tmp/vars.sh" > /dev/null
 
```

Comment:
Here after you have copied everything, I would suggest to delete the temp var file from the static worker.

Reply:
done